cli/dcoscli/service/main.py

```diff
@@ -305,4 +306,8 @@ def _log_marathon(follow, lines, ssh_config_file):
 
     emitter.publish(DefaultError("Running `{}`".format(cmd)))
 
+    # In the tests, we SIGKILL this process group, and the output
+    # doesn't get flushed unless we explicitly do so here.
+    sys.stderr.flush()
```

Comment:
I like exposing an optional flush parameter for `emitting.publish`. That is what `print` does in newer pythons. Why don't you think it is a good idea?